demos/kubernetes/README.md

```diff
@@ -9,7 +9,7 @@
 
     kubectl apply -f ./istio -n <ns>
 
-This will deploy istio discovery manager and istio mixer.
+This will deploy istio manager discovery and istio mixer.
```

Comment:
I think we can call it "discovery service". In reality, it combines what Envoy calls SDS, CDS, RDS, and in the future, LDS, which are all discovery services.